api/types/types.proto

```diff
@@ -3625,4 +3624,13 @@ message ConnectionDiagnosticV1 {
 message ConnectionDiagnosticSpecV1 {
     bool Success = 1 [ (gogoproto.jsontag) = "success" ];
     string Message = 2 [ (gogoproto.jsontag) = "message" ];
```

Comment:
What does the spec needs a message for?

Reply:
I added as a way to get back a summary message of whether the test result was a success or a failure.
Honestly, I'm not seeing any use case for this at this point.
We could probably remove it, no strong opinion here.

This field was not added in this PR, and the one that added the field was already backported to v10.
If needed, I'll open a PR to remove it.